test/bin/build_images.sh

```diff
@@ -361,7 +361,13 @@ do_group() {
     done
 
     if ${BUILD_INSTALLER} && ! ${COMPOSER_DRY_RUN}; then
-        for image_installer in "${groupdir}"/*.image-installer; do
+        image_installer_list="${groupdir}"/*.image-installer
+        # If a template arg was given, only build the image installer for the
+        # matching template.
+        if [ -n "${template_arg}" ]; then
+            image_installer_list="${template_arg%.toml}".image-installer
+        fi
+        for image_installer in "${image_installer_list}"; do
```

Comment:
Can we not alter the `for` loop pattern? There are edge cases on how wildcards are interpreted and expanded when passed through variables. 

As an alternative, inside the `for` loop, please just check if a `template_arg` was given and compare the file names. If names do not match, use `continue` to skip to the next loop iteration.